### PR TITLE
fix: docs spelling issues

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
I fixed several spelling issues in order to ensure a better understanding.
Br, Elias.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,13 +73,13 @@ The separation on "logic" and "server" crates is designed to separate functional
 For now, "server" crates contain gRPC and HTTP as the transport layer. 
 It was assumed, that users may want to implement their own APIs, for which the library with functional logic might be used.
 
-Crates that require database connection may also have additional sea-orm defined crates:
+Crates that require database connection may also have additional sea-orm-defined crates:
```
Comment (sevenzing) [inline README.md:76]: why? `sea-orm` is name of library

Comment (nnsW3) [reply README.md:76]: The hyphenation removes ambiguity and ensures the phrase is interpreted correctly as "crates that are defined by sea-orm."

Comment (bragov4ik) [reply README.md:76]: at the same time it adds another ambiguity 🫠 
smth like
```
`sea-orm`-defined
```
might be more clear

**Comment by nnsW3** [comment]
done

**Review by bragov4ik — APPROVED** [review]
lgtm

**Review by sevenzing — APPROVED** [review]

**Comment by sevenzing** [comment]
Thanks for contributing! Feel free to try your hand at coding tasks as well!

**Pull request merged**